README.md

```diff
@@ -1,100 +1,69 @@
+[uri_license]: https://opensource.org/licenses/MIT
 
-<!--
-
-Template variables to replace in ALL files:
-* __app_name__: Name of the application
-* __app_slug__: GitHub slug of the application
-* __app_description__: Application description
-
-After replacing all variables:
-* Search for any [TODO] and do the required operations to complete your project documentation and CI/CD.
-
--->
-
-[uri_license]: http://www.gnu.org/licenses/agpl.html
-[uri_license_image]: https://img.shields.io/badge/License-AGPL%20v3-blue.svg
+[uri_license_image]: https://img.shields.io/badge/license-MIT-blue
```

Comment:
No, that's not the license we want, nor the license that was already set in the template.